README.md

```diff
@@ -55,6 +55,7 @@ module "ocean-gcp-k8s" {
   # Credentials.
   cluster_name                      = var.cluster_name
   location                          = var.location
+  is_enabled                        = var.is_enabled
```

Comment:
please remove from the document, there are no other fields documented. it will create a confusion